src/main.cpp

```diff
@@ -262,6 +266,14 @@ int mainEmulatorHLE()
 	PPCCore_init();
 	// log Cemu startup info
 	infoLog_cemuStartup();
+	// enable gamemode (Linux-only library for standby prevention and some OS optimization)
+	#if BOOST_OS_LINUX
+		if (GetConfig().gamemode && gamemode_request_start() < 0)
```

Comment:
Short-circuiting controls whether gamemode is started or not. I think it's worth the extra lines to make it clearer what is happening.